app/assets/javascripts/application.js

```diff
@@ -17,6 +17,11 @@
 //= require numeral
 //= require miq_api
 //= require miq_angular_application
+//= require controllers/deployment/wizard-module
+//= require controllers/deployment/wizard/wizard-directive
+//= require controllers/deployment/wizard/wizard-buttons
+//= require controllers/deployment/wizard/wizard-step-directive
+//= require controllers/deployment/wizard/wizard-substep-directive
```

Comment:
Either way, please don't clutter application.js with these, since the next 3 lines will include them anyway. (And if you're having problems with module  not being included first, I suggest calling it wizard.js)..